src/jit/gentree.cpp

```diff
@@ -6308,9 +6308,9 @@ GenTreePtr          Compiler::gtCloneExpr(GenTree * tree,
         }
         copy->gtCall.gtRetClsHnd = tree->gtCall.gtRetClsHnd;
 
-#if defined(FEATURE_UNIX_AMD64_STRUCT_PASSING)
-        copy->gtCall.structDesc.CopyFrom(tree->gtCall.structDesc);
-#endif // defined(FEATURE_UNIX_AMD64_STRUCT_PASSING)  
+#ifdef FEATURE_UNIX_AMD64_STRUCT_PASSING
+        copy->gtCall.gtMultiRegReturnTypeDesc = tree->gtCall.gtMultiRegReturnTypeDesc;
+#endif
 
```

Comment:
I would prefer to have a "gtReturnTypeDesc" that is always present, and that describes the return type for any call, multi-reg or otherwise.